lib/meilisearch/task.rb

```diff
@@ -5,8 +5,13 @@
 
 module MeiliSearch
   class Task < HTTPRequest
-    def task_list
-      http_get '/tasks/'
+    ALLOWED_PARAMS = [:limit, :from, :next, :index_uid, :type, :status].freeze
```

Comment:
`next` is not a parameter, but a field in the response body only

Reply:
🤦 thanks for catching that!